src/API/Site/Controllers/MerchantCenter/SyncableProductsCountController.php

```diff
@@ -0,0 +1,141 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\BaseController;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\TransportMethods;
+use Automattic\WooCommerce\GoogleListingsAndAds\Jobs\JobRepository;
+use Automattic\WooCommerce\GoogleListingsAndAds\Jobs\UpdateSyncableProductsCount;
+use Automattic\WooCommerce\GoogleListingsAndAds\Options\TransientsInterface;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\RESTServer;
+use WP_REST_Request as Request;
+use WP_REST_Response as Response;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class SyncableProductsCountController
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\MerchantCenter
+ */
+class SyncableProductsCountController extends BaseController {
+
+	/**
+	 * @var TransientsInterface
+	 */
+	protected $transients;
+
+	/**
+	 * @var JobRepository
+	 */
+	protected $job_repository;
+
+	/**
+	 * SyncableProductsCountController constructor.
+	 *
+	 * @param RESTServer          $server
+	 * @param TransientsInterface $transients
+	 * @param JobRepository       $job_repository
+	 */
+	public function __construct( RESTServer $server, TransientsInterface $transients, JobRepository $job_repository ) {
+		parent::__construct( $server );
+		$this->transients     = $transients;
+		$this->job_repository = $job_repository;
+	}
+
+
+	/**
+	 * Registers the routes for the objects of the controller.
+	 */
+	public function register_routes() {
+		$this->register_route(
+			'mc/syncable-products-count',
+			[
+				[
+					'methods'             => TransportMethods::READABLE,
+					'callback'            => $this->get_syncable_products_count_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+				],
+			]
+		);
+
+		$this->register_route(
+			'mc/syncable-products-count',
+			[
+				[
+					'methods'             => TransportMethods::CREATABLE,
+					'callback'            => $this->update_syncable_products_count_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+				],
+			]
+		);
```

Comment:
Since the endpoint is the same we can combine these like we do [here](https://github.com/woocommerce/google-listings-and-ads/blob/2.1.2/src/API/Site/Controllers/Ads/CampaignController.php#L51-L68)

Reply:
Updated in 0094870.